Gemfile

```diff
@@ -7,8 +7,14 @@ gem 'rails', '~> 6.0.3', '>= 6.0.3.4'
 gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 4.1'
 gem 'bootsnap', '>= 1.4.2', require: false
+gem 'rack-cors', require: 'rack/cors'
 
 group :development, :test do
+  gem 'pry-rails'
+  gem 'pry-byebug'
```

Comment:
You don't need both so you can remove pry-byebug

Reply:
Deleted now!